rasterio/rio/edit_info.py

```diff
@@ -7,8 +7,10 @@
 
 from . import options
 import rasterio
-import rasterio.crs
+from rasterio._crs import CRS
```

Comment:
@jameshiebert should this be `from rasterio.crs import CRS` instead?  We shouldn't be pulling the Cython interface into places other than into `crs.py`